core/src/test/scripts/test_data/study_es_0/result_report.html

```diff
@@ -1279,6 +1279,218 @@ <h4 class="panel-title">data_mutations_extended.maf</h4>
         </div>
       </div>
 
+      <div class="panel panel-success">
```

Comment:
still a bit weird i think that the whole report is checked in as part of testing - we should prolly test specific things inside of the report after parsing it. But this works for now